yaml/presenter.nim

```diff
@@ -551,7 +551,13 @@ proc doPresent(s: var YamlStream, target: PresenterTarget,
         if levels.len > 0 and levels[levels.high] in
             [dFlowMapStart, dFlowMapValue]:
           raise newException(YamlPresenterJsonError,
-                             "Cannot have sequence as map key in JSON output!")
+
+
+
+
+
+
```

Comment:
PR looks good, but can you remove all these empty lines?

Reply:
ok, did that. (from the online editor)
btw, I'm new to github - btw how do I do that locally?

Reply:
@flyx ok now?